gtsam/inference/Ordering.h

```diff
@@ -71,9 +71,11 @@ class Ordering: public KeyVector {
    * @param keys The key vector to append to this ordering.
    * @return The ordering variable with appended keys.
    */
+  
   This& operator+=(KeyVector& keys);
 
   /// Check if key exists in ordering.
+  
```

Comment:
Please remove this newline since it interferes with doxygen's docs generation.

Reply:
done.